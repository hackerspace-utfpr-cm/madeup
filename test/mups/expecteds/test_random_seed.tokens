1(0-3) ID: seed
1(5-5) ASSIGN: =
1(7-8) INTEGER: 17
1(9-9) NEWLINE: \n
2(10-15) REPEAT: repeat
2(17-18) INTEGER: 30
2(19-19) NEWLINE: \n
3(22-26) ID: print
3(28-28) LEFT_PARENTHESIS: (
3(29-34) ID: random
3(36-36) INTEGER: 0
3(37-37) COMMA: ,
3(39-40) INTEGER: 17
3(41-41) RIGHT_PARENTHESIS: )
3(42-42) NEWLINE: \n
4(43-45) END: end
4(46-46) NEWLINE: \n
5(47-47) END_OF_FILE: END OF FILE
