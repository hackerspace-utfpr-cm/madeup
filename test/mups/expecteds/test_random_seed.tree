(block (define-variable seed (INTEGER 17)) (repeat (INTEGER 30) (block (call print (call random (INTEGER 0) (INTEGER 17))))))
