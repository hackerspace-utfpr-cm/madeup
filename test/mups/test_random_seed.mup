seed = 17
repeat 30
  print (random 0, 17)
end
