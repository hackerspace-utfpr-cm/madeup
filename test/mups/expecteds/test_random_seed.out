0
4
15
14
1
13
14
9
9
16
9
14
2
10
16
0
7
2
9
2
1
12
0
14
6
17
10
2
1
16
